requirements/framework.txt

```diff
@@ -8,3 +8,4 @@ pympler==1.0.1
 guppy3==3.1.2
 cron-schedule-triggers==0.0.11
 pytz==2019.3
+uvloop==0.17.0; sys_platform != 'win32'
```

Comment:
we have separate requirements texts for different architectures, why not make this a different file? Modifying the syntax of your requirements.txt seems.... nonstandard?

Reply:
That's a standard I've created in Python 10 years ago haha https://peps.python.org/pep-0508/#environment-markers
this is fully supported by pip and al

Reply:
see https://github.com/elastic/connectors-python/pull/197#issuecomment-1326799639